packages/admin/admin/src/common/readOnlyField/ReadOnlyDateField.tsx

```diff
@@ -0,0 +1,66 @@
+import { Calendar, Lock } from "@comet/admin-icons";
+import { css } from "@mui/material/styles";
+import { FunctionComponent, ReactNode } from "react";
+
+import { createComponentSlot } from "../../helpers/createComponentSlot";
+import { ReadOnlyFieldLabel } from "./ReadOnlyFieldLabel";
+
+export const ReadOnlyDateField: FunctionComponent<{
+    label?: ReactNode;
+    value?: Date | string | undefined;
+    className?: string;
+}> = ({ label, value, className }) => {
+    const date = value instanceof Date ? value : value ? new Date(value) : undefined;
+
+    return (
+        <Wrapper className={className}>
+            {label && <ReadOnlyFieldLabel label={label} />}
+            <Box>
+                <InnerBox>
+                    <Calendar />
+                    {date && date.toLocaleDateString()}
```

Comment:
We should use FormattedDate like we do for dataGridDateColumn: https://github.com/vivid-planet/comet/blob/main/packages/admin/admin/src/dataGrid/gridColumnTypes.tsx#L7.